.env

```diff
@@ -0,0 +1 @@
+ESLINT_NO_DEV_ERRORS=true
```

Comment:
https://stackoverflow.com/questions/64518226/my-create-react-app-is-failing-to-compile-due-to-eslint-error